content/TruenasApps/_index.md

```diff
@@ -134,10 +143,26 @@ Use to resolve issues where apps experiences issues where TrueNAS device is not
 Select the network option, or add additional options to resolve the network connection issues.
 {{< /hint >}}
 
-**Check for docker image updates** sets TrueNAS to check for docker image updates (default setting). 
+#### Installing NVIDIA Drivers
+Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must manually install drivers from the TrueNAS UI.
+
+If running TrueNAS 24.10 or higher:
+1. Go to **Apps > Installed** and click on the **Configuration**.
+
+2. Click on **Settings** to open the **Settings** configuration screen.
+
+3. Select **Install NVIDIA Drivers**, which is available to users with compatible GPUs.
+
+4. Select **Install NVIDIA Drivers**, and click **Save.**
+
+#### Monitoring for Image Updates
+
+Select **Check for docker image updates** (selected by default) to enable TrueNAS to periodically check for Docker image updates.
+This applies to all Docker images present on the system that may or may not for either catalog or custom applications.
```

Comment:
don't user "may" also something is missing in this sentence. incomplete thought?
```suggestion
This applies to all Docker images present on the system for either catalog or custom applications.
```

Reply:
ah that's what I get for copying over Stavros' explanation from slack